.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,41 @@
-<!--
-Please make sure you've read and understood our contributing guidelines;
-https://github.com/cri-o/cri-o/blob/master/CONTRIBUTING.md
-
-** Make sure all your commits include a signature generated with `git commit -s` **
+<!--  Thanks for sending a pull request!  Here are some tips for you:
 
-If this is a bug fix, make sure your description includes "fixes #xxxx", or
-"closes #xxxx"
-
-Please provide the following information:
+- If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
+- Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+- If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+- If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
```

Comment:
Is this all valid?  Just wondering if we should add a note that CRI-O too follows the k8s guidelines somewhere first?

Reply:
I'll add a note about that :+1: